.github/workflows/main.yml

```diff
@@ -9,15 +9,41 @@ on:
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
+    inputs:
+      sdk-ref:
+        description: 'sdk commit/tag/branch reference. Defaults to main.'
+        required: false
+        type: string
+        default: main
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
 jobs:
-
-  clippy:
-    name: Clippy
+  setup:
+    name: setup
+    runs-on: ubuntu-latest
+    outputs:
+      sdk-ref: ${{ inputs.sdk-ref || '0.2.7' }}
+      package-version: '0.2.7'
+    steps:
+      - run: echo "set pre-setup output variables"
+
+  build-packages:
+    needs: setup
+    name: build packages
+    uses: breez/breez-sdk/.github/workflows/publish-all-platforms.yml@main
```

Comment:
Which jobs from `publish-all-platforms.yml` will this be running?
Can't this trigger some publishing?

Reply:
No it can't trigger publishing, because the `publish` boolean is not available when calling that workflow. Also, this repository wouldn't have the required credentials if it were possible.

It runs the jobs to create the packages. These packages are archived. That's why we can use them here. The binaries are dummies to speed up compilation.